### PR TITLE
Update DG 

### PR DESCRIPTION
**Pull request by oscarsu97**
Include Scheduling feature in DG

### CODE REVIEW

File: docs/DevOps.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ ifdef::env-github[]
 :note-caption: :information_source:
 :warning-caption: :warning:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3/tree/master
+:repoURL: https://github.com/AY1920S1-CS2103T-T09-1/main
```
[the remark this answers is not in the document]
Comment (oscarsu97) [reply docs/DevOps.adoc]: alright thank you👍

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -243,6 +243,59 @@ _{Explain here how the data encryption feature will be implemented}_
 
 // end::dataencryption[]
 
+=== [Proposed] Schedule Activity feature
+
+*_Plan²travel_* allows user to schedule an activity from the activity list to a specified time slot of a day. They can be indicated
```
Comment (1nefootstep) [inline docs/DeveloperGuide.adoc]: Doesn't need to be in separate lines. You can enable wrapping in intellij to make it easier to write/read.

Comment (1nefootstep) [reply docs/DeveloperGuide.adoc]: Consider changing from `specified time slot of a day` to `specified time of a day`

Comment (1nefootstep) [reply docs/DeveloperGuide.adoc]: Consider changing from `They can be indicated...` to `This is accomplished...`

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -243,6 +243,59 @@ _{Explain here how the data encryption feature will be implemented}_
...
+by using the Schedule Command with index of the activity from activity list, index of the day from the list of days in Itinerary,
+start time and end time of the activity.
+
+Eg. `schedule activity ACTIVITY_INDEX st/START_TIME et/END_TIME d/DAY_INDEX`
+
+==== Current Implementation
+
+The keywords from the command given by the user is parsed using `ScheduleCommandParser` which converts the string
+variable of start time and end time into a LocalTime format and wraps activity index and day index  with an Index class.
+These are then passed on to the `ScheduleActivityCommand` for execution later on.
```
Comment (1nefootstep) [inline docs/DeveloperGuide.adoc]: passed to the

**Pull request merged**